File: Web/src/app/retrospective-tool/models/retro-info.model.ts
export class RetroInfoModel {
    /**
     * Constructor for RetroInfoDetails class
     * @param retroinfo_id
     * @param retroinfo_name
     * @param retroinfo_projectinfo_id
     * @param retroinfo_sprint
     * @param retroinfo_date
    */
    constructor(
        public retroinfo_id: number,
        public retroinfo_name: string,
        public retroinfo_projectinfo_id: number,
        public retroinfo_sprint: string,
        public retroinfo_date: string,
        public retroinfo_imagepath: string,
    ) { }
} 

File: Api/Models/RetroInfoModel.cs
namespace Retrospective.Application.API.Models
{
    using System;

    public class RetroInfoModel
    {
        public int retroinfo_id { get; set; }
        public string retroinfo_name { get; set; }
        public int retroinfo_projectinfo_id { get; set; }
        public string retroinfo_projectinfo_name { get; set; }
        public string retroinfo_sprint { get; set; }
        public string retroinfo_date { get; set; }
        public string retroinfo_imagepath { get; set; }
        public int retroinfo_imageinfo_id { get; set; }        
        public Nullable<bool> retroinfo_isdeleted { get; set; }
        public bool retroinfo_status { get; set; }
    }
}